### PR TITLE
feat: added payment and registration page

### PR DESCRIPTION
**Pull request by babitarit**

### CODE REVIEW

**Comment by starkjay21** [comment]
Please take care of commit message standards

**Comment by ashutosh-rath02** [comment]
@babitarit Please change your PR titile.

**Comment by ashutosh-rath02** [comment]
image
Please fix this.

**Review by Shurtu-gal — CHANGES_REQUESTED** [review]
@babitarit as can be seen [here](https://github.com/dscnitrourkela/project-udon/actions/runs/8320332803/job/22787358400?pr=19#step:7:16) `HeroText` has been changed to `HeroText1`.

Please modify it in your code.

**Comment by babitarit** [comment]
@Shurtu-gal after doing => HeroText has been changed to HeroText1 still deploy error exist

**Comment by Shurtu-gal** [comment]
> @Shurtu-gal after doing => HeroText has been changed to HeroText1 still deploy error exist

@babitarit I believe it is working.

File: src/components/shared/Register/Final.jsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+					}}>
+					Payment
+				</Heading>
+				<Paragraph variant='body3'>Send an email to recrourkela1998@gmail.com with a screenshot of your payment.</Paragraph>
+			</div>
+
+			<div className='container mx-auto px-4 lg:px-0'>
+				<div className='border border-solid border-black p-4 lg:w-1/2 lg:mx-auto'>
+					<Heading variant='h3' className='text-center my-8 lg:my-5'>
+						Payment method we have
```
Comment (ashutosh-rath02) [inline src/components/shared/Register/Final.jsx:37]: Move text to content

File: src/components/shared/Register/Register.jsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+				<ul className='list-disc mx-3 my-5'>
+					{Stepstoregister.map((item, index) => (
+						<li key={index}>
+							<Paragraph variant='body2' key={index}>
+								{item}
+							</Paragraph>
+						</li>
+					))}
+				</ul>
+			</div>
```
Comment (ashutosh-rath02) [inline src/components/shared/Register/Register.jsx:46]: `Payment.jsx` and `Register.jsx` seems to have same structure, create a reusable component and use it instead following good practices.

**Review by ashutosh-rath02 — CHANGES_REQUESTED** [review]

**Comment by Shurtu-gal** [comment]
closing as done

**Pull request closed without merging**